### DISCUSSION TITLE
Tài khoản EVN không hợp lệ. Xin kiểm tra lại Username và Password của bạn.

### DISCUSSION THREAD

**Discussion by vuong244** [trvqhuy/ha-evn#49] [General]
Lúc cấu hình khi chọn ngày hoá đơn là ngày 1 thì nó báo là không kết nối được server, nhấn nhiều lần thì nó báo: Tài khoản EVN không hợp lệ. Xin kiểm tra lại Username và Password của bạn.
Còn để ngày mặc định là 24 thì được

**Comment by tuanna0896** [comment]
Bác ở miền Bắc à

**Reply by vuong244** [reply]
em ở miền nam ạ
